Python/ogusa/SS.py

```diff
@@ -625,7 +625,7 @@ def run_steady_state(income_tax_parameters, ss_parameters, iterative_params, get
                 output_dir, "Saved_moments/SS_init_solutions.pkl")
             pickle.dump(outputs, open(ss_init_path, "wb"))
             function_to_minimize_X = lambda x: function_to_minimize(
-                x, chi_params, income_tax_params, ss_parameters, iterative_params, omega_SS, rho, lambdas, tau_bq, e, output_dir)
+                x, chi_params, income_tax_parameters, ss_parameters, iterative_params, omega_SS, rho, lambdas, tau_bq, e, output_dir)
```

Comment:
`income_tax_params` is not a valid variable name in this function, changed to `income_tax_parameters`.